features/users/user_rename.feature

```diff
@@ -163,3 +163,13 @@ Feature:
       And I should see "Hi, newusername"
     When I follow "Series"
     Then I should see "Best Series by newusername"
+
+    Scenario: Changing the username from a forbidden name to non-forbidden
+      Given I am logged in as "forbidden" with password "password"
+        And the user name "forbidden" is on the forbidden list
```

Comment:
Can we tweak this a little so the name is on the forbidden list before we log in? (The spec that shows you can save changes if you're not editing the user means this scenario _should_ be fine, but I think it's better to explicitly cover it in the feature.)

```
Given the following activated user exists
  | login     |  etc |
  | forbidden |      |
  And the user name "forbidden" is on the forbidden list
When I log in as etc
```